src/plugins/components/layout/centered-container.js

```diff
@@ -2,36 +2,46 @@
  * Centered container.
  */
 module.exports = function () {
-  return function ({ addComponents, theme }) {
+  return function ({ addUtilities, theme }) {
```

Comment:
Why the change to Utility here?

Reply:
> Use the addUtilities function to register simple static utilities that don’t support user-provided values

My belief is this matches the criteria. But I honestly think they accomplish the same things. :shrug:

Reply:
https://tailwindcss.com/docs/adding-custom-styles

```
The base layer is for things like reset rules or default styles applied to plain HTML elements.
The components layer is for class-based styles that you want to be able to override with utilities.
The utilities layer is for small, single-purpose classes that should always take precedence over any other styles.
```

I believe we chose the components option for this one so that other utility classes would win if the developer chose to add different behaviours later.

Reply:
ahh. interesting! yeah i agree. I didn't realize that it would add them at different layer